src/rendering/RenderUtil.cc

```diff
@@ -162,6 +163,12 @@ class ignition::gazebo::RenderUtilPrivate
   public: std::vector<std::tuple<Entity, msgs::ParticleEmitter, Entity>>
       newParticleEmitters;
 
+  /// \brief New particle emitter commands to be requested.
+  /// The elements in the tuple are: [0] entity id of the particle emitter to
+  /// update, [1] particle emitter, [2] entity id of the particle emitter cmd.
+  public: std::vector<std::tuple<Entity, msgs::ParticleEmitter, Entity>>
+      newParticleEmittersCmd;
```

Comment:
Higher-level design question: would it be possible to make `particleEmitterCmd` a component of the particle emitter entity it is referring to, instead of having it as a separate entity? It seems to me like the command(s) for updating an emitter should be a component of the emitter to be updated since the commands are associated with the original emitter entity.

Reply:
a07c2ba